doltpy/cli/tests/fixtures.py

```diff
@@ -0,0 +1,26 @@
+import os
+from doltpy.cli.dolt import Dolt
+import pytest
+import shutil
+from doltpy.cli.tests.helpers import get_repo_path_tmp_path
+
+
+@pytest.fixture
```

Comment:
nit: scope

Reply:
What do you mean?

Reply:
yeah, knowing what you're creating every function, package, module etc is helpful

Reply:
I actually use `conftest.py` to pull these into the appropriate package, is this in-lieu of that mechanism?

Reply:
https://docs.pytest.org/en/stable/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session